### PR TITLE
Neutron driver: convert all boxed comments to normal open source style

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

**Comment by neiljerram** [comment]
Anyone care to review this?

**Comment by neiljerram** [comment]
I've also now added PEP8 compliance to this PR.

File: calico/openstack/mech_calico.py [hunk 1]
```diff
@@ -181,100 +162,73 @@ def initialize(self):
...
+                    # filter, so we filter the results ourselves instead.
                     LOG.info("Query Neutron DB...")
-                    ports = []
-                    for port in self.db.get_ports(db_context):
-                        if (port['binding:host_id'] == rq['hostname'] and
-                            self._port_is_endpoint_port(port)):
-                            ports.append(port)
+                    ports = [port for port in self.db.get_ports(db_context)
+                             if (port['binding:host_id'] == rq['hostname'] and
+                                 self._port_is_endpoint_port(port))]
```
Comment (Lukasa) [inline calico/openstack/mech_calico.py]: :star: :star2:

File: calico/openstack/mech_calico.py [hunk 2]
```diff
@@ -344,7 +288,8 @@ def _ensure_socket_to_felix_peer(self, hostname, db_context):
                                                {'agent_type': AGENT_TYPE_FELIX,
                                                 'binary': '',
                                                 'host': hostname,
-                                                'topic': constants.L2_AGENT_TOPIC,
+                                                'topic':
+                                                constants.L2_AGENT_TOPIC,
```
Comment (Lukasa) [inline calico/openstack/mech_calico.py]: I would want to indent this one level to make it clearer that it's a value, not a key.

**Comment by fasaxc** [comment]
:+1:

**Pull request merged**